bin/github-review.sh

```diff
@@ -68,9 +68,9 @@ if [[ -n "$(gh pr diff $pr --name-only | grep -v -e 'src/assets/v'$version'/.*\.
 fi
 
 echo "Checking changes to assets files"
-fileLineChangeJSON=$(gh pr view $pr --json files) # This line fetches the JSON output of the files changed in the PR
-diffPatchSeparated=$(gh pr diff $pr | grep -E '^[+-] ' | base64) # Sending multiple lines as a string through base64 encoding
-npm run verifyChanges -s -- --fileLineChange "$fileLineChangeJSON" --diffPatchSeparated "$diffPatchSeparated"  --verbose
+gh pr diff $pr > .temp-github-review.diff # This line fetches the diff output of the PR
+npm run review:diff -s -- --diffPatchFileName .temp-github-review.diff --verbose
+rm .temp-github-review.diff
```

Comment:
If you want to cleanup the file more reliably after running the script, then you should use `trap`. Personally, I would say "screw it" and just add it to the project's .gitignore.

Reply:
Why not both 😅 TIL `trap`.